### PR TITLE
Update development oauth

### PR DESCRIPTION
**Pull request by jdconrad89**
@drod1000 @Laszlo-JFLMTCO @mollybrown  The Oauth for the development on localhost is working now. Just a quick update on what was done. in the gemfile I updated 
```gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census.git"``` to 
```gem 'omniauth-census', git: "https://github.com/NZenitram/census_staging_oauth"```

Visit https://github.com/NZenitram/census_staging_oauth and follow the walkthrough of setting up the gem thin. You don't need to install it as it is already in our Gemfile. I don't know if it's the norm but for Daniel and I it didn't work on chrome but I found that after this I was able to go through the oauth in development on Safari. 

For reference sake so we know what we are working with I took a picture of what the information looks like coming in from census so we can utilize this to help populate things where we are able to.
 
image

@neight-allen

### CODE REVIEW

**Comment by drod1000** [comment]
@jdconrad89 Good to know regarding the chrome vs safari issue. This will definitely help us move quicker from here on out! Also, did you mean to merge this into master or development?

**Comment by jdconrad89** [comment]
@drod1000 thanks for catching that! It's been switched to merge into development. Also it might work in chrome now, I know that Nate made a few changes to the codebase of census to support localhost.

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GIT
-  remote: https://github.com/turingschool-projects/omniauth-census.git
-  revision: 3d28a5040e79eae620c3550b90e94aeed8234571
+  remote: https://github.com/NZenitram/census_staging_oauth
+  revision: 082f9c376dd85e6d65415f8d1dac0e054971433c
```
Comment (Laszlo-JFLMTCO) [inline Gemfile.lock:3]: @jdconrad89 As we discussed, let's plan to make it environment dependent, so we will not have to change it each time we want to evaluate in a test, staging or production environments.

FYI: @mollybrown @drod1000

**Comment by Laszlo-JFLMTCO** [comment]
@jdconrad89 Many thanks for working through this topic. This has been a very complex one, figuring out all the details along the way. Good job!!!
Also as we discussed I added a comment to document that we will need to make the `oauth` related settings environment dependent to avoid manually updating it every time we want to evaluate the implementation in a different environment.
I am OK to merge this PR.

FYI @drod1000 @mollybrown

File: Gemfile.lock (hunk 1 above)
Comment (neight-allen) [reply Gemfile.lock:3]: 👍

**Pull request merged**